neuraxle/base.py

```diff
@@ -952,36 +953,16 @@ def inverse_transform(self, processed_outputs):
         :func:`~neuraxle.base.BaseStep.mutate` or :func:`~neuraxle.base.BaseStep.reverse` can be called to change the default transform behavior :
 
         .. code-block:: python
-            class MultiplyByN(NonFittableMixin, BaseStep):
-                def __init__(self, multiply_by):
-                    NonFittableMixin.__init__(self)
-                    BaseStep.__init__(
-                        self,
-                        hyperparams=HyperparameterSamples({
-                            'multiply_by': multiply_by
-                        })
-                    )
-
-                def transform(self, data_inputs):
-                    return data_inputs * self.hyperparams['multiply_by']
-
-                def inverse_transform(self, processed_outputs):
-                    return data_inputs / self.hyperparams['multiply_by']
 
-            p = Pipeline([
-                MultiplyByN(multiply_by=2)
-            ])
+            p = Pipeline([MultiplyBy()])
```

Comment:
It was okay or better to have `multiply_by=2` instead of the default `multiply_by=1`, but thats a detail